src/domain/home/hero/hero.module.scss

```diff
@@ -15,6 +15,9 @@
     background-size: 100% auto;
     display: grid;
     padding-bottom: $largeMargin;
+    @include respond-above(lg) {
+        min-height: 35vh;
```

Comment:
We might need to increase it a bit, 40 and 45 is better i think

Reply:
Looked at it with Anniina, I increased the value in the developer console until she said "stop, now it looks good"

Reply:
Ok